src/main/java/mate/jdbc/dao/CarDao.java

```diff
@@ -0,0 +1,19 @@
+package mate.jdbc.dao;
+
+import java.util.List;
+import java.util.Optional;
+import mate.jdbc.model.Car;
+
+public interface CarDao {
+    Car create(Car car);
+
+    Optional<Car> get(Long id);
+
+    List<Car> getAllByDriver();
```

Comment:
lets rename this method to just getAll()
We already have getAllByDriver method

Reply:
Fixed.